tests/test_script_loader.py

```diff
@@ -22,40 +22,54 @@
 import asyncio
 import os
 import unittest
+import warnings
 
 import SALPY_ScriptLoader
+import SALPY_Script
 import salobj
-import scriptrunner
+import scriptloader
 
 
 class ScriptLoaderTestCase(unittest.TestCase):
     def setUp(self):
+        salobj.test_utils.set_random_lsst_dds_domain()
         self.datadir = os.path.abspath(os.path.join(os.path.dirname(__file__), "data"))
         standardpath = os.path.join(self.datadir, "standard")
         externalpath = os.path.join(self.datadir, "external")
-        self.loader = scriptrunner.ScriptLoader(standardpath=standardpath, externalpath=externalpath)
-        self.remote = salobj.Remote(SALPY_ScriptLoader, "ScriptLoader:0")
+        self.loader = scriptloader.ScriptLoader(standardpath=standardpath, externalpath=externalpath)
+        self.remote = salobj.Remote(SALPY_ScriptLoader)
+        self.process = None
+
+    def tearDown(self):
+        nkilled = self.loader.model.terminate_all()
+        if nkilled > 0:
+            warnings.warn(f"Killed {nkilled} subprocesses")
 
     def test_load(self):
         async def doit():
             load_data = self.remote.cmd_load.DataType()
+            load_data.is_standard = False
             load_data.path = "script1"
-            load_data.is_standard = True
-            script_info_data = self.remote.evt_script_info.DataType()
-            info_coro = self.remote.evt_script_info.next(script_info_data)
-            id_ack = await self.remote.cmd_load.start(load_data, timeout=5)
+            load_data.config = "wait_time: 1"
+            info_coro = self.remote.evt_script_info.next(timeout=2)
```

Comment:
`next` as a method name could be a bit confusing because of the `next()` Python built-in function.

Reply:
I agree, but I think the name is clear enough that I'm inclined to keep it. If you disagree strongly feel free to file a ticket against salobj